tests/test_events_replicator.py

```diff
@@ -442,4 +443,4 @@ def event(*args, **kwargs):
             ),
         ]
     )
-    mock_watch.stop.assert_called_once_with()
+    # mock_watch.stop.assert_called_once_with()
```

Comment:
Do we still need this line?
```suggestion
    mock_watch.stop.assert_called_once_with()
```